### DISCUSSION TITLE
Physical hop buttons

### DISCUSSION THREAD

**Discussion by IromTypo** [runelite/runelite#17184] [Ideas]
Could an option be added to one click hop down or up a world with 2 overlay arrow buttons? Would make shopscape a lot easier!

**Comment by ShaunDreclin** [comment]
You can already do this with a keybind, do you mean like a button in the interface to do it?

**Reply by IromTypo** [reply]
Yes, so you don't have to use your keyboard to hop! Going between keyboard and mouse is annoying lol

Sorry for the late reply!
